theme/icons.go

```diff
@@ -652,13 +665,16 @@ func (res *ErrorThemedResource) Original() fyne.Resource {
 	return res.source
 }
 
-// PrimaryThemedResource is a resource wrapper that will return a version of the resource with the main color changed
-// to the theme primary color.
+// PrimaryThemedResource is a resource wrapper used to represent a version of a resource.
+// It changes colors based on the active theme.
+// This is currently achieved using XML attributes.
+// This is designed for use with Icons of 24px x 24px.
 type PrimaryThemedResource struct {
 	source fyne.Resource
 }
 
-// NewPrimaryThemedResource creates a resource that adapts to the primary color for the current theme.
+// NewThemedResource creates a resource that adapts to the current theme primary color.
```

Comment:
This line is incorrect. It is missing `Primary`.
```suggestion
// NewPrimaryThemedResource creates a resource that adapts to the current theme primary color.
```

Reply:
Thanks @Jacalz I made this change locally yesterday, but the generated docs are not what I was looking for. I mentioned in the Slack channel; I may close this out, or look for an alternative approach.